core/src/zeit/content/article/article.py

```diff
@@ -560,3 +571,26 @@ def get_dependencies(self):
         if audio_refs:
             return audio_refs.items
         return ()
+
+
+@grok.subscribe(
+    zeit.content.article.interfaces.IArticle, zeit.cms.checkout.interfaces.IBeforeCheckinEvent
+)
+def update_recipes_of_article(context, event):
+    if not FEATURE_TOGGLES.find('wcm_19_store_recipes_in_storage'):
+        return
+    if context.genre not in ['rezept', 'rezept-vorstellung']:
+        return
+    recipes = list(context.body.filter_values(zeit.content.modules.interfaces.IRecipeList))
```

Comment:
```suggestion
    recipes = context.body.filter_values(zeit.content.modules.interfaces.IRecipeList)
```

Es wird ja gleich dann eh nur 1x drüber iteriert, muss man also nicht vorher ein zweites Mal drüberlaufen um es in eine Liste zu packen.